pkg/splunk/enterprise/deployer.go

```diff
@@ -192,14 +192,12 @@ func ApplyDeployer(ctx context.Context, client splcommon.ControllerClient, cr *e
 
 // SHC connected to the deployer adds its owner reference on the deployer CR
 // getShcConnDeployer extracts the SHC owner reference, retrieve and return the SHC CR
-func getShcConnDeployer(ctx context.Context, c splcommon.ControllerClient, deployerCr splcommon.MetaObject) (*enterpriseApi.SearchHeadCluster, error) {
+var getShcConnDeployer = func(ctx context.Context, c splcommon.ControllerClient, deployerCr splcommon.MetaObject) (*enterpriseApi.SearchHeadCluster, error) {
```

Comment:
nit: you may want to use a better name? for `getShcConnDeployer`

Reply:
I was trying to go for `Get the SHC connected to the deployer`. Any suggestions here?